packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -5310,65 +5310,6 @@ export default class Meeting extends StatelessWebexPlugin {
     return this.updateMedia({mediaSettings: this.mediaProperties.mediaDirection});
   }
 
-  /**
-   * Sets the quality level of all meeting media (incoming/outgoing)
-   * @param {String} level {LOW|MEDIUM|HIGH}
-   * @returns {Promise}
-   */
-  setMeetingQuality(level) {
```

Comment:
Removed setMeetingQuality method as well as its test code becasue we are not using this.
Please confirm. @arun3528